AppflowSDK.podspec

```diff
@@ -0,0 +1,53 @@
+#
+# Be sure to run `pod lib lint AppflowSDK.podspec' to ensure this is a
+# valid spec before submitting.
+#
+# Any lines starting with a # are optional, but their use is encouraged
+# To learn more about a Podspec see https://guides.cocoapods.org/syntax/podspec.html
+#
+
+Pod::Spec.new do |s|
+  s.name             = 'AppflowSDK'
+  s.version          = '1.0.0'
+  s.summary          = 'A short description of AppflowSDK.'
+
+# This description is used to generate tags and improve search results.
+#   * Think: What does it do? Why did you write it? What is the focus?
+#   * Try to keep it short, snappy and to the point.
+#   * Write the description between the DESC delimiters below.
+#   * Finally, don't worry about the indent, CocoaPods strips it!
+
+  s.description      = <<-DESC
+TODO: Add long description of the pod here.
+                       DESC
+
+  s.homepage         = 'https://github.com/appflow-ai/AppflowSDK.git'
+  s.license          = { :type => 'MIT', :file => 'LICENSE' }
+  s.author           = { 'appflow.ai' => 'sdk-dev@appflow.ai' }
+  s.source           = { :git => 'https://github.com/appflow-ai/AppflowSDK.git', :tag => s.version.to_s }
+  # s.social_media_url = 'https://twitter.com/<TWITTER_USERNAME>'
+
+  s.ios.deployment_target = '11.0'
+
+  s.vendored_frameworks  = "AppflowExample/Appflow/AppflowSDK.xcframework "
+
+  #s.source_files = 'AppflowExample/Appflow/*.h'
+  
+  # s.resource_bundles = {
+  #   'AppflowSDK' => ['AppflowSDK/Assets/*.png']
+  # }
+
+  s.dependency 'gRPC-ProtoRPC', '~>1.31.0'
+  s.dependency 'FMDB' 
+  s.dependency 'Protobuf','~>3.13.0'
+  s.dependency 'PLCrashReporter'
+  s.dependency 'Founq_KeychainIDFA'
```

Comment:
After checked the podfile of the example app, it seems that all our users need to add this dependency list to their podfile. So question:  Can we make this dependency list as less as possible?